.github/workflows/updatecli.d/bump-golang-7.17.yml

```diff
@@ -13,6 +13,7 @@ scms:
       token: '{{ requiredEnv "GITHUB_TOKEN" }}'
       branch: "7.17"
       commitusingapi: true
+      force: false
```

Comment:
Is this what would prevent the force-push that closes the PR?

Reply:
Correct, that's the alternative, already added to the description, but see this link:
- https://github.com/updatecli/updatecli/issues/2211#issuecomment-2147430601

This solves the problem partially, as it will not create any new updates with a new version either updating the existing open PR or creating a new of if an existing open PR exists. 

I need to see if using an old version of updatecli, as mentioned at https://github.com/updatecli/updatecli/issues/2211#issuecomment-2159939390, reverts to the original approach (see https://github.com/elastic/beats/pull/41241/commits).